### DISCUSSION TITLE
Quiz App - warisa

### DISCUSSION THREAD

**Discussion by warisaa** [sst-inc/help#34] [Q&A - iOS] [answered]
<img width="1440" alt="Screenshot 2021-08-23 at 16 44 20" src="https://user-images.githubusercontent.com/87971971/130417846-b2f3d1c3-6375-4bdc-b22c-aa8c6ec68f9c.png">
<img width="899" alt="Screenshot 2021-08-23 at 16 45 43" src="https://user-images.githubusercontent.com/87971971/130418114-87108735-d173-431e-b053-067641a3b196.png">
<img width="899" alt="Screenshot 2021-08-23 at 16 45 54" src="https://user-images.githubusercontent.com/87971971/130418126-fc1efac4-23b4-41f9-bac1-7f3787b5a709.png">

what's wrong with mine??
thx🙏

**Comment by Ethan-Chew** [comment]
@granwyntan 
File for the Xcode project.
[Quiz App.zip](https://github.com/sst-inc/help/files/7036918/Quiz.App.zip)

**Comment by Ethan-Chew** [comment, marked as answer]
Hi @warisaa, can you try disconnecting the questionLabel and reconnecting the questionLabel?
